src/test/moment/zones.js

```diff
@@ -461,9 +461,27 @@ test('timezone format', function (assert) {
 
 test('parse zone without a timezone', function (assert) {
     test.expectedDeprecations();
-    var m = moment.parseZone('2016-02-01T00:00:00');
+    var m1 = moment.parseZone('2016-02-01T00:00:00');
+    var m2 = moment.parseZone('2016-02-01T00:00:00Z');
+    var m3 = moment.parseZone('2016-02-01T00:00:00+00:00'); //Someone might argue this is not necessary, you could even argue that is wrong being here.
+    var m4 = moment.parseZone('2016-02-01T00:00:00+0000'); //Someone might argue this is not necessary, you could even argue that is wrong being here.
```

Comment:
Agreed, I'd remove the comments and keep those tests.

I would, however, move your three new checks into their own test case -- named something like "parse zone with UTC input" -- since UTC is a valid ~~timezone~~ offset and this test case says it's testing the parser when that information isn't part of the input string.  In other words, putting your new tests in this test block implies that we would expect the old (incorrect) behavior, so I'd split them into their own function.